src/parser/warrior/fury/modules/azerite/SimmeringRage.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import { formatNumber, formatPercentage, formatThousands } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import TraitStatisticBox from 'interface/others/TraitStatisticBox';
+import SPELLS from 'common/SPELLS/index';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+import Events from 'parser/core/Events';
+
+
+const simmeringRageDamage = traits => Object.values(traits).reduce((obj, rank) => {
+    const [ damage ] = calculateAzeriteEffects(SPELLS.SIMMERING_RAGE.id, rank);
+    obj.damage += damage;
+    return damage;
+  }, {
+    damage: 0,
+  });
+
+class SimmeringRage extends Analyzer {
+    rampage = [SPELLS.RAMPAGE_1.id, SPELLS.RAMPAGE_2.id, SPELLS.RAMPAGE_3.id, SPELLS.RAMPAGE_4.id];
```

Comment:
Things like this should be moved to constants above the class and renamed to fit that convention:
`const RAMPAGE = [...]`